pom.xml

```diff
@@ -930,6 +930,8 @@
                     <java>
                         <excludes>
                             <exclude>src/main/java/org/apache/seatunnel/antlr4/generated/*.*</exclude>
+                            <exclude>**/TimeplusSinkFactory.java</exclude>
+                            <exclude>**/TimeplusSinkWriter.java</exclude>
```

Comment:
Why?

Reply:
Good catch. I guess I added them during development to avoid checkstyle warning. I have removed these 2 lines